### PR TITLE
[10.0] [ADD] partner_phone_extension

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Partner Phone Number Extension
===============================

This module adds a phone number extension field in partner form view.

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot

File: partner_phone_extension/README.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+===============================
+Partner Phone Number Extension
+===============================
+
+This module adds a phone number extension field in partner form view.
+
+
```
Comment (AaronHForgeFlow) [inline partner_phone_extension/README.rst]: One blank line is enough

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM. Checked also on runbot. :+1:

**Review by AaronHForgeFlow — APPROVED** [review]

File: partner_phone_extension/__init__.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013-2014 Savoir-faire Linux
+#   (<http://www.savoirfairelinux.com>).
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
```
Comment (yajo) [inline partner_phone_extension/__init__.py]: Man! So much copyright holders for 1 line of code, isn't it?
🔥 this (see https://github.com/OCA/maintainer-tools/pull/260)

File: partner_phone_extension/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013-2014 Savoir-faire Linux
+#   (<http://www.savoirfairelinux.com>).
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
```
Comment (yajo) [inline partner_phone_extension/models/__init__.py]: 🔥

File: partner_phone_extension/models/res_partner.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+#   (<http://www.serpentcs.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    extension = fields.Char('Extension', help="Phone Number Extension.")
```
Comment (yajo) [inline partner_phone_extension/models/res_partner.py]: I guess the name should be `phone_extension`, adding `oldname="extension"` to the field definition.

**Review by yajo — CHANGES_REQUESTED** [review]

File: partner_phone_extension/models/res_partner.py [hunk 5]
```diff
@@ -13,4 +12,5 @@
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    extension = fields.Char('Extension', help="Phone Number Extension.")
+    extension = fields.Char('Phone Extension', oldname="extension",
```
Comment (yajo) [inline partner_phone_extension/models/res_partner.py]: But call the field `phone_extension`

File: partner_phone_extension/models/res_partner.py [hunk 6]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
Comment (yajo) [inline partner_phone_extension/models/res_partner.py]: This precise line you have to keep it, I was talking about the copyright ones.

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by AaronHForgeFlow** [comment]
Hi @nikul-serpentcs, you forgot to change the field name in the view

File: partner_phone_extension/views/res_partner_view.xml [hunk 7]
```diff
@@ -22,7 +22,7 @@
                 <field name="extension"/>
```
Comment (AaronHForgeFlow) [inline partner_phone_extension/views/res_partner_view.xml]: @nikul-serpentcs Also here please

**Comment by AaronHForgeFlow** [comment]
@Yajo can you review this again?

**Comment by LoisRForgeFlow** [comment]
Please, @nikul-serpentcs squash the commits of the migration into just one.

**Comment by nikul-serpentcs** [comment]
@lreficent squashed the commits

**Review by yajo — APPROVED** [review]

**Pull request merged**